src/log.ts

```diff
@@ -0,0 +1,690 @@
+/* eslint @typescript-eslint/ban-types: "off", @typescript-eslint/no-explicit-any: "off" */
+import { H, h, frag, styled } from "./virtualdom";
+
+export class HtmlLogger {
```

Comment:
```suggestion
export class HTMLLogger {
```
I'd prefer this to be `HTMLLogger` to match with `HTMLElement` for consistency.

Reply:
Done.